src/planning/plan/replan-queue.ts

```diff
@@ -14,7 +14,7 @@ import {now} from '../../platform/date-web.js';
 import {PlanProducer, Trigger} from './plan-producer.js';
 import {Dictionary} from '../../runtime/hot.js';
 
-const defaultDefaultReplanDelayMs = 3000;
+const defaultDefaultReplanDelayMs = 300;
```

Comment:
Double-default naming seems strange but I realize it was already this way.

Reply:
Lol, I didn't even notice that. Is it on purpose @mmandlis?

Reply:
no, this is by mistake, sorry.

Reply:
I can see how it's the `default` for `options.defaultReplanDelayMs`. Maybe the problem is that it should be `options.replayDelayMs`? Is there ever a time where it does something other than the _default_?

Reply:
I guess `nextReplanDelayMs` is calculated (so the answer to my question is _yes_). I'm going to tweak the names anyway to remove one 'default'.

Reply:
oh, i see it now, sorry for the bad answer as well as poor naming :(
it never does anything other than default, but i wanted it to have to option to...

Reply:
Lol, stop worrying so much! Frankly I think your naming was actually precise. I just made it less precise but hopefully more readable. There's no obvious right answer IMO.